File: hackerrank/easy/0026-find-digits/js/solution.mjs
// solution.mjs

export function findDigits(n) {
  const digits = String(n).split('').map(Number);
  let count = 0;
  for (const d of digits) {
    if (d !== 0 && n % d === 0) count++;
  }
  return count;
}

// --- demo run (from sample) ---
console.log(findDigits(12));   // 2
console.log(findDigits(1012)); // 3

File: hackerrank/easy/0026-find-digits/python/solution.py
## solution.py

def find_digits(n: int) -> int:
    count = 0
    for d in str(n):
        digit = int(d)
        if digit != 0 and n % digit == 0:
            count += 1
    return count


# --- demo run (from sample) ---
if __name__ == "__main__":
    print(find_digits(12))    # 2
    print(find_digits(1012))  # 3